services/core-web/src/components/noticeOfWork/applications/review/ReviewNOWApplication.js

```diff
@@ -220,20 +230,44 @@ export const ReviewNOWApplication = (props) => {
             Do you need to build a road, create stream crossings or other surface disturbance that
             will not be on your tenure?**
           </div>
-          <Field id="" name="" component={RenderRadioButtons} disabled />
+          <Field
+            id="has_surface_disturbance_outside_tenure"
+            name="has_surface_disturbance_outside_tenure"
+            component={RenderRadioButtons}
+            disabled={props.isViewMode}
+          />
         </Col>
         <Col md={12} sm={24}>
-          <div className="field-title">Access presently gated**</div>
-          <Field id="" name="" component={RenderRadioButtons} disabled />
+          <div className="field-title--light">
+            Please provide the type and authorization numbers for each access authorization
+            application or exemption to use the road(s).**
```

Comment:
Do we still need the '**' in the label names? I thought this was to indicate that we weren't recording them, but now we are?